boards/rapidsilicon/virgo_tc_eval_soc_proto/virgo_tc_eval_soc_proto.dts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024 Rapid Silicon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ /dts-v1/;
+
+ #include <rapidsilicon/rapidsi_virgo.dtsi>
+ /
+ {
+	 model = "Rapid Silicon Virgo Test Chip Evaluation";
```

Comment:
This will only apply to virgo soc proto since the clocks are divided by 10. For virgo tc we can create a revision in the same folder with specific dts for increased clock values.